### DISCUSSION TITLE
The program isn't working.

### DISCUSSION THREAD

**Discussion by chandramouli007-coder** [Alby0n/Masked-Face-Recognition#8] [Q&A]
Hey. I have implemented the project as said in the documentation and ran all the snippets in the google drive. I have uploaded the zip file containing the images of people (7 different people) to be recognized and the video footage in the format of mp4 of only one subject. The program is creating several frames and is not finding the name of the person and the program runs indefinitely. What should I do to solve this error. Your assistance and guidance would really be useful. 

Thanking you
Chandra Mouli. S

**Comment by Alby0n** [comment]
Could you share me the screenshots of the UI and Colab last cell

**Reply by chandramouli007-coder** [reply]
![image](https://user-images.githubusercontent.com/71344864/184467970-9a2a230f-9a30-46c3-b2e6-838ddf2535ce.png)
![image](https://user-images.githubusercontent.com/71344864/184467980-8821799d-f3b2-4ef0-ae14-ce5dc32b04e7.png)
![image](https://user-images.githubusercontent.com/71344864/184467984-99fec1ad-be72-47a7-ad01-cc24a7bedfaf.png)
![image](https://user-images.githubusercontent.com/71344864/184467988-f7b12258-07e5-4932-8cf0-ab9d53250a06.png)
![image](https://user-images.githubusercontent.com/71344864/184468014-246faf99-ed8b-47ce-b99c-71faecc9e8d6.png)

![image](https://user-images.githubusercontent.com/71344864/184468030-aaa0b497-f86a-4304-9285-bfb118e26422.png)

![image](https://user-images.githubusercontent.com/71344864/184467997-86f482e7-0bd3-4c59-8281-16e9b3b0fe0b.png)
![image](https://user-images.githubusercontent.com/71344864/184468003-9493aa2f-4f5c-413e-8b64-31af9f4a7883.png)

**Reply by Alby0n** [reply]
Did you directly zipped the photos together or made a folder with photos into zip ?

**Reply by chandramouli007-coder** [reply]
I had a folder with the images and I zipped it.

**Reply by Alby0n** [reply]
Zip the images together , not as a folder . Also try without mask and check too . You might have to do some parameter tuning as you are uploading high quality video that of which doesnt look cctv quality .

**Reply by chandramouli007-coder** [reply]
Sir do you mean I should zip every pic and upload rather as a folder?

**Reply by Alby0n** [reply]
You can select all the images you want and directly turn it into zip

**Reply by chandramouli007-coder** [reply]
Ok I will try it and let you know if I face difficulties. And thanks for replying this soon.

**Reply by Alby0n** [reply]
😊

**Reply by chandramouli007-coder** [reply]
![image](https://user-images.githubusercontent.com/71344864/184468428-f4675feb-a850-4e16-8af0-388bcdfa9c6c.png)
![image](https://user-images.githubusercontent.com/71344864/184468433-368f298d-bad2-4c00-989a-ab90037baeb3.png)
![image](https://user-images.githubusercontent.com/71344864/184468435-bc036be5-3dcb-4871-b177-042989d62b83.png)


The program seems to be working now but it isn't providing the photograph in the database and the name given in the output 3 box "Mouli 20137006.jpg" is not correct it should have been "Vimal 20137054.jpg".

What should I do. I am working on a project to implement masked face recognition and mask detection project. I have referred some content on Git and YouTube but have only made a decent progress. Can you please guide me how I can implement both of them. It would really be useful.

**Reply by Alby0n** [reply]
I think , I didn't merge the revised code with those changes . I am on vacation right now , will upload these revisions in the coming week. If you want to implement the things you are asking for just check out the detection model part , that is where frames are being returned to the UI  and also make the appropriate changes in the UI code as well.

**Reply by chandramouli007-coder** [reply]
Thank you. And can you please refer me sources for me to learn from to implement the changes and can you tell me how do I become an expert in this field like you. And if you would be kind enough to upload the revised version it would be easy to cross check. Please suggest me good resources and I am really grateful for your help. Thank you.

**Reply by Alby0n** [reply]
Reading documentations is important. Always check github for the trending pre-trained models. For detection I used caffe , you can google for articles such as face detection using caffe . For recognition I used a Siamese neural network , for better understanding there are courses on Coursera by DEEPLEARNING.ai. Keep reading and tweaking open source codes , you will figure out everything eventually. HAPPY CODING!!

**Reply by chandramouli007-coder** [reply]
Thank you and can I approach you for any further doubts regarding future projects.
